site2/docs/functions-develop.md

```diff
@@ -656,6 +656,185 @@ $ bin/pulsar-admin functions create \
 
 All logs produced by `LoggingFunction` above can be accessed via the `persistent://public/default/logging-function-logs` topic.
 
+#### Customize Function log level
+Additionally, you can use the XML file, `functions_log4j2.xml`, to customize the function log level. 
+To customize the function log level, create or update `functions_log4j2.xml` in your Pulsar conf directory (for example, `/etc/pulsar/` on bare-metal, or `/pulsar/conf` on Kubernetes) to contain contents such as:
```

Comment:
@eolivelli I added a snippet on the Kubernetes path. Is this what you were looking for?

Reply:
Perfect. Thanks